inbc-program/inbc/command/command.py

```diff
@@ -129,7 +129,7 @@ def invoke_update(self, args: Any) -> None:
 
         @param args: arguments passed to command-line tool.
         """
-        super()._send_manifest(args, RESTART_CHANNEL)
+        super()._send_manifest(args, RESTART_CMD_CHANNEL)
 
     def search_response(self, payload: Any) -> None:
         """Search for keywords in response message
```

Comment:
**<h2>$\color{#FFA500}Improvements: Code Maintainability$</h2>**
**Suggestion:** The search_response method in the Command class is marked as an abstract method but provides a default implementation. This could be misleading. If a default implementation is desired, consider removing the @abstractmethod decorator and documenting the default behavior. Otherwise, subclasses should provide their own implementation without a default in the base class.
**NOTE:** Ensure that the suggested code is comprehensive. This serves as an example of the code you could use. The user should verify it after accepting the suggestion.
```suggestion
def search_response(self, payload: Any) -> None:
    if self._cmd_type != "query":
        if search_keyword(payload, ["SUCCESSFUL", '"status": 200']):
            self.terminate_operation(COMMAND_SUCCESS, InbcCode.SUCCESS.value)
        else:
            self.terminate_operation(COMMAND_FAIL, InbcCode.FAIL.value)
```